src/main.js

```diff
@@ -0,0 +1,145 @@
+import './styles/main.scss';
+
+const weatherApp = () => {
+  let responseData = {};
+  const form = document.getElementById('form');
+
+  const displayError = () => {
+    const alert = document.querySelector('.alert');
+    alert.classList.add('show');
+  };
+
+  const parseData = (response) => ({
+    cityName: response.name,
+    countryCode: response.sys.country,
+    description: response.weather[0].description,
+    icon: response.weather[0].icon,
+    temperature: response.main.temp,
+    feelsTemp: response.main.feels_like,
+    minTemp: response.main.temp_min,
+    maxTemp: response.main.temp_max,
+    clouds: response.clouds.all,
+    wind: response.wind.speed,
```

Comment:
* [ ] ES6 - use object destructuring to get the values from an object. This way you can avoid repeating a lot of code. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/destructuring_requirement.png) from Pathwright)
```javascript
const { name, sys, weather, main, clouds, wind } = response
```